netsim/providers/__init__.py

```diff
@@ -55,10 +54,23 @@ def get_template_path(self) -> str:
     return 'templates/provider/' + self.provider
 
   def get_full_template_path(self) -> str:
-    return str(_files.get_moddir()) + '/' + self.get_template_path()
+    return os.path.join(str(_files.get_moddir()), self.get_template_path())
```

Comment:
Nitpicking (since you decided to get into this): IIRC, the get_moddir() returns a Path object, so you could use Path operators and then convert the whole thing to string